app/models/post.js

```diff
@@ -68,6 +69,15 @@ export default Model.extend(ValidationEngine, {
 
     isPublished: equal('status', 'published'),
     isDraft: equal('status', 'draft'),
+    isScheduled: equal('status', 'scheduled'),
+
+    // Checks every second, if we reached the scheduled date
+    timeScheduled: computed('publishedAt', 'clock.second', function () {
```

Comment:
I'm not sure this is a good place to handle this - it will mean we have a task running every second for every blog post that is in the store.

I think perhaps a posts list component would be a better place for it - that way it can track only the posts that are rendered on the screen and should be able to set up tracking only for those posts which are scheduled.

Leave it as-is for now, this is more of a note for an area to re-visit later.